internal/datastore/postgres/revisions.go

```diff
@@ -27,12 +30,17 @@ const (
 	//   %[2] Relationship tuple transaction table
```

Comment:
in the query description, might be worth adding some info about the `xmin`. It's really detailed right now, but doesn't mention the xmin at all.